erpnext/public/js/call_popup/call_popup.js

```diff
@@ -140,6 +140,15 @@ class CallPopup {
 			}, {
 				'fieldtype': 'Section Break',
 				'hide_border': 1,
+			}, {
+				'fieldname': 'type_of_call',
+				'label': 'Type Of Call',
+				'fieldtype': 'Select',
+				'options': '\nFrappe Cloud Queries/Plan\nEnterprise Plans\nPartnership\nSupport\nBilling/Renewal\nOpen source / Junk',
```

Comment:
👀

"Frappe Cloud Queries" or other options here are not really relevant for all telephony users... better to have a link field instead.